components/data-feed-service/habitat/default.toml

```diff
@@ -6,6 +6,8 @@ asset_page_size = 100
 reports_page_size = 1000
 node_batch_size = 100
 updated_nodes_only = true
+disable_cidr_filter = true
```

Comment:
this is a cool new feature! 
it also reminds me to ask, are there docs somewhere for all this functionality?

Reply:
Not yet, the task is on the board and will get them done when ready to remove the feature flag